storage/aptosdb/src/db/include/aptosdb_writer.rs

```diff
@@ -489,6 +483,7 @@ impl AptosDB {
         Ok(root_hash)
     }
 
+    #[allow(dead_code)]
```

Comment:
Is this dead code? AI complains😄

Reply:
We still want to keep this auxiliary data field and function to write it for other use-cases.